sympy/series/fourier.py

```diff
@@ -90,6 +91,37 @@ def _find_x(func):
     return sympify((x, start, stop))
 
 
+def finite_check(f, x):
+    def check_fx(exprs, x):
+        return x not in exprs.free_symbols
+
+    def check_sincos(expr, x):
+        if type(expr) == sin or type(expr) == cos:
+            a = Wild('a', properties=[lambda k: x not in k.free_symbols, ])
+            b = Wild('b', properties=[lambda k: x not in k.free_symbols or k == S.Zero, ])
+
+            sincos_args = expr.args[0]
+
+            if sincos_args.match(a * x + b) is not None:
+                return True
+            else:
+                return False
+
+    expr = sincos_to_sum(TR2(TR1(f)))
+    res_expr = S.Zero
+    add_coeff = expr.as_coeff_add()
+    res_expr += add_coeff[0]
+
+    for s in add_coeff[1]:
+        mul_coeffs = s.as_coeff_mul()[1]
+        for t in mul_coeffs:
+            if not(check_fx(t, x) or check_sincos(t, x)):
```

Comment:
Put a space after `not`

```suggestion
            if not (check_fx(t, x) or check_sincos(t, x)):
```